app/src/main/java/org/liberty/android/fantastischmemo/ui/QuizLauncherDialogFragment.java

```diff
@@ -505,6 +576,18 @@ public void onFocusChange(View v, boolean hasFocus) {
             }
         };
 
+    View.OnFocusChangeListener timerInputListener =
+            new View.OnFocusChangeListener() {
+
+                @Override
+                public void onFocusChange(View v, boolean hasFocus) {
+                    if (hasFocus == false) {
+                        timerModeSetTimeEdit.setText("" + chosenTime);
```

Comment:
Why do you need "" + chosenTime and not just chosenTime?

Reply:
I did it to be consistent with the other listeners tbh lol. I can change it if you want